### PR TITLE
test-process-util: Migrate to new assertion macros

### PR DESCRIPTION
**Pull request by daandemeyer**

### CODE REVIEW

File: src/test/test-process-util.c [hunk 1]
```diff
@@ -520,49 +525,49 @@ TEST(pid_get_cmdline_harder) {
 #define EXPECT1p "foo $'\\'bar\\'' $'\"bar$\"' $'x y z' $'!``'"
 #define EXPECT1v STRV_MAKE("foo", "'bar'", "\"bar$\"", "x y z", "!``")
 
-        assert_se(lseek(fd, SEEK_SET, 0) == 0);
-        assert_se(write(fd, CMDLINE1, sizeof CMDLINE1) == sizeof CMDLINE1);
-        assert_se(ftruncate(fd, sizeof CMDLINE1) == 0);
+        ASSERT_OK_ZERO(lseek(fd, SEEK_SET, 0));
```
Comment (yuwata) [inline src/test/test-process-util.c]: ASSERT_OK_ZERO_ERRNO() ?

File: src/test/test-process-util.c [hunk 2]
```diff
@@ -520,49 +525,49 @@ TEST(pid_get_cmdline_harder) {
...
+        ASSERT_OK_EQ_ERRNO(write(fd, CMDLINE1, sizeof(CMDLINE1)), (ssize_t) sizeof(CMDLINE1));
+        ASSERT_OK_ZERO(ftruncate(fd, sizeof(CMDLINE1)));
```
Comment (yuwata) [inline src/test/test-process-util.c]: Also here.

File: src/test/test-process-util.c [hunk 3]
```diff
@@ -520,49 +525,49 @@ TEST(pid_get_cmdline_harder) {
...
 #define EXPECT2  "foo \"\\001\\002\\003\""
 #define EXPECT2p "foo $'\\001\\002\\003'"
 #define EXPECT2v STRV_MAKE("foo", "\1\2\3")
 
-        assert_se(lseek(fd, SEEK_SET, 0) == 0);
-        assert_se(write(fd, CMDLINE2, sizeof CMDLINE2) == sizeof CMDLINE2);
-        assert_se(ftruncate(fd, sizeof CMDLINE2) == 0);
+        ASSERT_OK_ZERO(lseek(fd, SEEK_SET, 0));
+        ASSERT_OK_EQ_ERRNO(write(fd, CMDLINE2, sizeof(CMDLINE2)), (ssize_t) sizeof(CMDLINE2));
+        ASSERT_OK_ZERO(ftruncate(fd, sizeof CMDLINE2));
```
Comment (yuwata) [inline src/test/test-process-util.c]: Also here.

File: src/test/test-process-util.c [hunk 4]
```diff
@@ -831,30 +847,31 @@ TEST(get_process_ppid) {
...
         int fd = PTR_TO_FD(p);
         char x;
 
         /* let main thread know we are ready */
-        assert_se(write(fd, &(const char) { 'x' }, 1) == 1);
+        ASSERT_OK_EQ_ERRNO(write(fd, &(const char) { 'x' }, 1), 1);
 
         /* wait for the main thread to tell us to shut down */
-        assert_se(read(fd, &x, 1) == 1);
+        ASSERT_OK_EQ(read(fd, &x, 1), 1);
```
Comment (yuwata) [inline src/test/test-process-util.c]: ASSERT_OK_EQ_ERRNO() ?

**Review by yuwata — APPROVED** [review]
LGTM. Several comments.

**Comment by DaanDeMeyer** [comment]
@yuwata Added one more commit, PTAL

**Review by yuwata — APPROVED** [review]

**Pull request merged**